### PR TITLE
🍰 2019/kw24/fix_imported_img_urls

### PR DESCRIPTION
**Pull request by Tirokk**
> [ulfgebhardt](https://github.com/ulfgebhardt) **Authored by [ulfgebhardt](https://github.com/ulfgebhardt)**
_<time datetime="2019-06-14T12:07:22Z" title="Friday, June 14th 2019, 2:07:22 pm +02:00">Jun 14, 2019</time>_
_Merged <time datetime="2019-06-21T21:41:32Z" title="Friday, June 21st 2019, 11:41:32 pm +02:00">Jun 21, 2019</time>_
---

## 🍰 Pullrequest

Fixed URLs on import to make them relative.
This works only for `api-alpha` URLs.

### Issues

- relates #692 

### Todo

- [X] None

### CODE REVIEW

File: deployment/legacy-migration/maintenance-worker/migration/neo4j/badges.cql [hunk 1]
```diff
@@ -45,7 +45,7 @@ MERGE(b:Badge {id: badge._id["$oid"]})
 ON CREATE SET
 b.key       = badge.key,
 b.type      = badge.type,
-b.icon      = badge.image.path,
+b.icon      = replace(badge.image.path, 'https://api-alpha.human-connection.org', ''),
```
Comment (Tirokk) [inline deployment/legacy-migration/maintenance-worker/migration/neo4j/badges.cql:48]: > [roschaefer](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-06-14T19:07:45Z" title="Friday, June 14th 2019, 9:07:45 pm +02:00">Jun 14, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -45,7 +45,7 @@ MERGE(b:Badge {id: badge._id["$oid"]})
 ON CREATE SET
 b.key       = badge.key,
 b.type      = badge.type,
-b.icon      = badge.image.path,
+b.icon      = substring(badge.image.path, 38),
```


@ulfgebhardt I find it quite dangerous, how do you know that it's always the same first part of the URL?

File: deployment/legacy-migration/maintenance-worker/migration/neo4j/contributions.cql [hunk 2]
```diff
@@ -28,7 +28,7 @@
 [?]     unique: true, // Unique value is not enforced in Nitro?
 [-]     index: true
       },
-[ ]   type: {
+[ ]   type: { // db.getCollection('contributions').distinct('type') -> 'DELETED', 'cando', 'post'
```
Comment (Tirokk) [inline deployment/legacy-migration/maintenance-worker/migration/neo4j/contributions.cql:31]: > [roschaefer](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-06-14T19:08:35Z" title="Friday, June 14th 2019, 9:08:35 pm +02:00">Jun 14, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -28,7 +28,7 @@
 [?]     unique: true, // Unique value is not enforced in Nitro?
 [-]     index: true
       },
-[ ]   type: {
+[ ]   type: { // db.getCollection('contributions').distinct('type') -> 'DELETED', 'cando', 'post'
```


Why is `DELETED` a type :thinking: ?

File: deployment/legacy-migration/maintenance-worker/migration/neo4j/badges.cql (hunk 1 above)
Comment (Tirokk) [reply deployment/legacy-migration/maintenance-worker/migration/neo4j/badges.cql:48]: > [roschaefer](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-06-14T19:10:53Z" title="Friday, June 14th 2019, 9:10:53 pm +02:00">Jun 14, 2019</time>_

---


Use [replace](https://neo4j.com/docs/cypher-manual/current/functions/string/#functions-replace).

File: deployment/legacy-migration/maintenance-worker/migration/neo4j/contributions.cql (hunk 2 above)
Comment (Tirokk) [reply deployment/legacy-migration/maintenance-worker/migration/neo4j/contributions.cql:31]: > [ulfgebhardt](https://github.com/ulfgebhardt) **Authored by [ulfgebhardt](https://github.com/ulfgebhardt)**
_<time datetime="2019-06-17T10:16:33Z" title="Monday, June 17th 2019, 12:16:33 pm +02:00">Jun 17, 2019</time>_

---


I cannot tell you - its just sure it is there

**Comment by Tirokk** [comment]
> [codecov[bot]](https://github.com/apps/codecov) **Authored by [codecov[bot]](https://github.com/apps/codecov)**
_<time datetime="2019-06-18T15:43:02Z" title="Tuesday, June 18th 2019, 5:43:02 pm +02:00">Jun 18, 2019</time>_

---


# [Codecov](https://codecov.io/gh/Human-Connection/Human-Connection/pull/829?src=pr&el=h1) Report
> :exclamation: No coverage uploaded for pull request base (`master@53ce729`). [Click here to learn what that means](https://docs.codecov.io/docs/error-reference#section-missing-base-commit).
> The diff coverage is `n/a`.

File: webapp/package.json [hunk 3]
```diff
@@ -29,6 +29,7 @@
       "!**/?(*.)+(spec|test).js?(x)"
     ],
     "coverageReporters": [
+      "text",
```
Comment (Tirokk) [inline webapp/package.json:32]: > [roschaefer](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-06-21T21:08:03Z" title="Friday, June 21st 2019, 11:08:03 pm +02:00">Jun 21, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -29,6 +29,7 @@
       "!**/?(*.)+(spec|test).js?(x)"
     ],
     "coverageReporters": [
+      "text",
```


Unrelated?

**Review by Mogge — APPROVED** [review]
aww yeah

**Pull request closed without merging**